### DISCUSSION TITLE
社会实践作业

### DISCUSSION THREAD

**Discussion by naming-is-difficult** [share121/inter-knot#5000] [委托]
1.巴黎 m奥运会里印象最深的运动员与比赛

2.运动员夺冠时的感受（不限中国

3.巴黎奥运会的亮点与该改进的地方

4.如何评价 m运动员的精神品质（不限中国

5.遇到困难时你如何应用运动员们的品质（不想答不答

邦邦窝罢😭😭😭

**Comment by HKLHaoBin** [comment]
你为啥发了5次？
